### ISSUE TITLE
ZOWE Explorer. Http error when trying to log in using base profile (Unable to log in. z/OSMF REST API Error: Rest API failure with HTTP(S) status 405)

### ISSUE THREAD

**Issue by jguillaumes** [zowe/vscode-extension-for-zowe#1808]
**Describe the bug**

Since upgrading to V2, when I try to use a base profile I get the following error at login time:

Unable to log in. z/OSMF REST API Error:
Rest API failure with HTTP(S) status 405

**To Reproduce**

- Clean up ZOWE configuration (delete .zowe folders, delete local zowe configuration at project level)
- Create a team configuration file:

(snippets follow):

 ```
      "zosmf": {
            "type": "zosmf",
            "properties": {
                "host": "zowes4.lacaixa.es",
                "port": 7554,
                "basePath": "api/v1",
                "encoding": "1047",
                "rejectUnauthorized": false
            },
            "secure": []
.
.
.
      "base": {
            "type": "base",
            "properties": {
                "host": "zowes4.lacaixa.es",
                "port": 7554,
                "rejectUnauthorized": false,
                "user": "T88640G"
            },
            "secure": [
                "user",
                "password"
            ]
```

- Try to login ("Log in to authentication service")
- Boom

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

- OS: Windows 10 + Ubuntu Linux 20.04 running under WSL2
- Zowe Explorer Version: 2.0.1
- (Optional) Are you using Secure Credential Store? No

**Additional context**

**Comment by zFernand0** [comment]
This may be related:
- #1750

**Comment by jguillaumes** [comment]
Hello,

Found this in the ZOWE APIML logs:

`IZUSVR WARN  (o.s.w.s.PageNotFound) No handler found for POST /gateway/auth/login[\n]"[\n]"[\n]"[\n]"[\n]"[\n]"[\n]"[\n]"[\n]"[\n]"`

This causes the http:405 status.

It looks like the basePath is missing (it should be /api/v1/gateway/blahblahblah)

**Comment by zFernand0** [comment]
I believe it depends on the version of Zowe APIML that you have installed.
However, there was an enhancement introduced (I think on v1) to support both paths.
@balhar-jakub may have more info on this 😋

**Comment by JTonda** [comment]
Closed as issue was resolved in #1339
